packages/overlay/test/index.ts

```diff
@@ -429,6 +420,20 @@ export const runOverlayTriggerTests = (type: string): void => {
                     'inner click content available at point'
                 ).to.be.true;
 
+                // Why does this make the test pass in Chromium? 🤷🏻‍♂️
```

Comment:
How did you figure this out?

Reply:
In the next test, it was previously using `preseSpace()` (though I also don't know why). When the test started to fail, I switched that to an actual keyboard interaction and it started to pass, so I adopted the approach here, too.

Reply:
Makes sense to me.... interesting. Thanks for explaining.